### DISCUSSION TITLE
Docs rewrite in Vitepress to match Vue docs?

### DISCUSSION THREAD

**Discussion by PatrikBird** [DamianOsipiuk/vue-query#153] [Ideas]
Hey @DamianOsipiuk, thanks for creating vue-query! 

You already gave it a thought to rewrite the docs when https://github.com/vuejs/vitepress/projects/1 is v1.0? 

You may ask why? Here are my opinionated reasons:
- increases trust in the lib as it's the same theme etc as Vue's docs
- current vue-query docs are not easy on the eye

Let me know what you think.

cheers, Patrik

**Comment by DamianOsipiuk** [comment]
@PatrikBird
Hi,
I do not feel currently like rewriting the docs.
I like the current super simple setup, that does not require any build process.
Also i think that dark mode is a prerequisite, at least for me, cause i tend to work in dark environment and light themes are actually not easy for my eyes 😉 And i think vitepress does not have one currently.

I do not say though that it would not be a thing in the future. Just that right now i do not think is feasible to work on it.
